addons/l10n_it_edi_doi/models/account_chart_template.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = 'account.chart.template'
+
+    def _load(self, company):
+        res = super()._load(company)
+        template = company.chart_template_id
+        if template == self.env.ref('l10n_it.l10n_it_chart_template_generic'):
+            doi_tax = self.env.ref(f'l10n_it_edi_doi.{company.id}_00di')
```

Comment:
```suggestion
            doi_tax = self.env.ref(f'l10n_it_edi_doi.{company.id}_00di', raise_if_not_found=False)
            if doi_tax:
```
Just to be safe (same for the fiscal position) :innocent: